src/appleseed.shaders/src/maya/as_maya_anisotropic.osl

```diff
@@ -269,7 +269,7 @@ shader as_maya_anisotropic
                 );
         }
 
-        if (transparency)
+        if (max(transparency) > 0.0)
```

Comment:
While technically correct, `max()` is (in theory) more computationally intensive than necessary; `any(color) > 0.0` would have been enough.

This is purely a theoretical remark since of course the max of three values can be computed in 1 cycle or less using SSE, and it's likely that the OSL compiler does just that...